### DISCUSSION TITLE
Is it possible to visualize Decision trees in River?

### DISCUSSION THREAD

**Discussion by nehalAggarwal** [online-ml/river#521] [Q&A] [answered]
Hello,

I am using Extremely Fast Decision Trees for a dataset and they seem to be performing too well. I thought it would be great if I could see how the tree is splitting the data. 

So is there a way I can visualize the tree like it is done in Sklearn trees (graphviz, plot_tree etc.) module? 
Or is there any way that the model can be seen in a human-understandable way?

Thank you.

**Comment by smastelini** [comment, marked as answer]
Hi @nehalAggarwal. You can use the `draw` function. :)

Soon we'll merge a general guide showing how to do decision tree model inspection, among other functionalities (#488).

**Reply by smastelini** [reply]
You can check the new entry in the user guidelines that shows how to inspect and handle Hoeffding Trees.
